build/source/engine/coupled_em.f90

```diff
@@ -64,9 +64,8 @@ module coupled_em_module
 
 ! look-up values for the numerical method
 USE mDecisions_module,only:         &
- iterative,                         &      ! iterative
- nonIterative,                      &      ! non-iterative
- iterSurfEnergyBal                         ! iterate only on the surface energy balance
+ bEuler,                            &      ! home-grown backward Euler solution with long time steps
```

Comment:
Are these changes part of this PR that incorporates timing outputs at the HRU level, or are these part of another PR?